### DISCUSSION TITLE
GitHub Education Pack approved 5 days back but benefits not provided yet.

### DISCUSSION THREAD

**Discussion by amanimam295** [community/community#136591] [GitHub Education]
### Hi everyone,

Hi, I had applied for the GitHub student Education Pack 5 days back, where it showed "your application has been approved and your benefits will be provided soon". 5 days are over, yet it's still showing "your benefits will be provided soon". Kindly help.

**Comment by davevad93** [comment]
Hi @amanimam295 ,

As stated in the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) if you haven't received your benefits within 5 days you should raise a discussion here. However, most of the time the benefits become available without the need for direct assistance from the education team.

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @amanimam295 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849
